generators/client/files-angular.js

```diff
@@ -123,9 +123,6 @@ const files = {
         {
             path: ANGULAR_DIR,
             templates: [
-                // entities
-                'entities/_index.ts',
-                'entities/_entity.module.ts',
```

Comment:
Fully agree with entity.module.ts, not so sure about index.ts because it's going to be huge.